Packs/CommonScripts/Scripts/EncodeToAscii/EncodeToAscii.py

```diff
@@ -0,0 +1,26 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+''' MAIN FUNCTION '''
+
+
+def main():
+    # Grab 'data' from Demisto Arguments
+    data = demisto.args()['data']
+
+    # Encode the data, ignoring characters
+    try:
+        encoded_data = data.encode('ascii', 'ignore').decode("utf-8")
+    except Exception as e:
+        return_error(f'Failed to encoding the data.\nError:\n{str(e)}')
```

Comment:
```suggestion
        return_error(f'Failed to encode the data.\nError:\n{str(e)}')
```
or as before: `There was an error encoding the data.`